src/decorator.ts

```diff
@@ -55,10 +55,10 @@ export abstract class Decorator extends PathTransformVisitor {
     return not(isLibrary);
   }
 
-  abstract get name(): string;
+  abstract get testDecorator(): (node: DecoratorNode) => boolean;
```

Comment:
Maybe I would suggest that this is a normal method rather than a get function. Also, the function name is a bit weird. I prefer decoratorMatcher to keep it consistent with the sourceFilter style.